glide.lock

```diff
@@ -66,7 +66,7 @@ imports:
   - peer
   - transport
 - name: k8s.io/kubernetes
-  version: e72f26a3fff4f521d1da9f52b3ce82d4d0d5a401
+  version: v1.5.0-alpha.1
```

Comment:
This will be overwritten on each `glide install` to hash value.
I'll fix that in my PR.